### DISCUSSION TITLE
项目中的playground放入本地项目之后，如何配置所谓的路由~

### DISCUSSION THREAD

**Discussion by dhp1215** [alibaba/formily#2554] [Q&A 2.x] [answered]
1、如何配置所谓的路由~
2、如何建出带请求分页的表格
3、是否有一个例子，举例去集成进去配置1 or 2 的问题。

**Comment by janryWang** [comment, marked as answer]
1. designable提供的就是一堆组件，你平时怎么用组件的，现在就怎么用
2. 需要自己扩展，不知道怎么扩展的，看视频，看源码，维护者没时间也没必要手把手教写代码
3. 同样属于手把手教程，这种要么自己研究，要么等designable慢慢把文档体系建立起来

**Reply by dhp1215** [reply]
就像我直接配置出一个列表，我要放到某个路由或者某个地方，不的指定吗，目前是怎么做的

**Reply by janryWang** [reply]
你要想清楚，designable生成的产物就是个json，你的渲染引擎用formily也好，用自己写的也好，都只是渲染这个json，路由都是属于外围的事情，不属于designable要解决的问题范畴，这个是你自己要处理的
